pkg/storage/tsdb/config.go

```diff
@@ -90,3 +91,26 @@ func (cfg *Config) Validate() error {
 
 	return nil
 }
+
+// BucketStoreConfig holds the config information for Bucket Stores used by the querier
+type BucketStoreConfig struct {
+	Dir                  string `yaml:"dir"`
+	IndexCacheSizeBytes  uint64 `yaml:"index_cache_size_bytes"`
+	MaxChunkPoolBytes    uint64 `yaml:"max_chunk_pool_bytes"`
+	MaxSampleCount       uint64 `yaml:"max_sample_count"`
+	MaxConcurrent        int    `yaml:"max_concurrent"`
+	DebugLogging         bool   `yaml:"debug_logging"`
+	BlockSyncConcurrency int    `yaml:"block_sync_concurrency"`
+}
+
+// RegisterFlags registers the BucketStore flags
+func (cfg *BucketStoreConfig) RegisterFlags(f *flag.FlagSet) {
```

Comment:
By convention, all CLI flags description should start with an upper case and end with `.`